### DISCUSSION TITLE
No apto para principiantes

### DISCUSSION THREAD

**Discussion by albertoperopero** [skills/.github#189] [Ideas]
Considérenme principiante en el área, en el tema de GitHUb me costo seguir los pasos, no hay correlación entre las instrucciones y los resultados.

**Comment by valeria-heredia** [comment]
a mi me pareció excelente y super buen explicado

**Comment by aryamcc** [comment]
pensé lo mismo, incluso el día de seguir el tutorial al leer la primera instrucción dije no podre hacerlo!!! sin embargo lo deje por ese día y regrese al dia siguiente y creo que hasta ahora vamos bien!!
